src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -924,15 +924,37 @@ namespace
         }
     }
 
+    bool is_light_emitting_material(Mtl* mtl)
+    {
+        IAppleseedMtl* appleseed_mtl =
+            static_cast<IAppleseedMtl*>(mtl->GetInterface(IAppleseedMtl::interface_id()));
+        if (appleseed_mtl != nullptr && appleseed_mtl->can_emit_light())
+            return true;
+        else
+        {
+            const int submtl_count = mtl->NumSubMtls();
+            if (submtl_count > 0)
+            {
+                for (int i = 0; i < submtl_count; ++i)
+                {
+                    Mtl* sub_mtl = mtl->GetSubMtl(i);
+                    if (sub_mtl != nullptr)
+                    {
+                        if (is_light_emitting_material(sub_mtl))
+                            return true;
+                    }
+                }
+            }
+        }
+        return false;
+    }
+
     bool has_light_emitting_materials(const MaterialMap& material_map)
     {
         for (const auto& entry : material_map)
         {
             Mtl* mtl = entry.first;
-            IAppleseedMtl* appleseed_mtl =
-                static_cast<IAppleseedMtl*>(mtl->GetInterface(IAppleseedMtl::interface_id()));
-            if (appleseed_mtl->can_emit_light())
-                return true;
+            return is_light_emitting_material(mtl);
```

Comment:
That doesn't look right. It should most likely be:

    if (is_light_emitting_material(mtl))
        return true;